backend/services/inventory/tests/tests/test_mgmt_v2_filters_attributes.py

```diff
@@ -11,40 +11,38 @@
 #    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
-from common import (
-    inventory_attributes,
-    management_client,
-    management_client_v2,
-    internal_client,
-    clean_db,
-    mongo,
-)
-
-import bravado
+
 import pytest
 
+import openapi_client as oas
+
+from client import make_authenticated_client
+
 
 @pytest.mark.usefixtures("clean_db")
 class TestGetAttributes:
     def test_get_attributes(
         self,
-        management_client,
-        management_client_v2,
-        internal_client,
         inventory_attributes,
     ):
+        auth_client = make_authenticated_client(is_device=False)
+        internal_client = oas.InventoryInternalApi()
+        management_client = oas.InventoryManagementApi(auth_client)
+        management_client_v2 = oas.InventoryManagementV2Api(auth_client)
         # NOTE: bravado request and response checks are enabled for management API v2 client
```

Comment:
```suggestion
        # NOTE: bravado has left the tests.
```

Reply:
Right. Should probably have `grep -ri bravado tests/`